com.regnosys.rosetta/src/com/regnosys/rosetta/Rosetta.xtext

```diff
@@ -678,8 +678,9 @@ RosettaRuleReference:
 	'[' 'ruleReference' reportingRule=[RosettaBlueprint] ']'	
 ;
 
+// @Compat: make ':' mandatory at January 16
```

Comment:
I'm thinking of making a comment such as this one for any feature that needs transitioning, so we don't forget about it.

Reply:
Yeah, it's a good idea